cross-platform/react-app/src/frontend/Screens/ModelScreen.tsx

```diff
@@ -420,6 +442,9 @@ export function ModelScreen(props: ModelScreenProps) {
         {additionalComponents}
         {tabsAndPanelsAPI.renderTabBarAndPanels()}
       </MobileUiContent >
+      <UiStateStorageHandler>
+        <ConfigurableUiContent />
+      </UiStateStorageHandler>
```

Comment:
Why is this being added here?

Reply:
`ConfigurableUiContent` is where modal dialogs come. Without it, they don't show up in the UI at all. It must be inside `UiStateStorageHandler` to work.

Reply:
Fixed (along with improving the corresponding css comment).